src/Entity/ContextInterface.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\SdkContracts\Entity;
+
+use SprykerSdk\SdkContracts\Violation\ViolationReportInterface;
+
+interface ContextInterface
+{
+    /**
+     * @var string
+     */
+    public const DEFAULT_STAGE = 'default';
+
+    /**
+     * @var array<string>
+     */
+    public const DEFAULT_STAGES = [self::DEFAULT_STAGE];
+
+    /**
+     * @var int
+     */
+    public const SUCCESS_EXIT_CODE = 0;
+
+    /**
+     * @var int
+     */
+    public const FAILURE_EXIT_CODE = 1;
+
+    /**
+     * @return array<\SprykerSdk\SdkContracts\Entity\PlaceholderInterface>
+     */
+    public function getRequiredPlaceholders(): array;
+
+    /**
+     * @param \SprykerSdk\SdkContracts\Entity\PlaceholderInterface $placeholder
+     *
+     * @return void
+     */
+    public function addRequiredPlaceholder(PlaceholderInterface $placeholder): void;
+
+    /**
+     * @param array<\SprykerSdk\SdkContracts\Entity\PlaceholderInterface> $requiredPlaceholders
+     *
+     * @return void
+     */
+    public function setRequiredPlaceholders(array $requiredPlaceholders): void;
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function getResolvedValues(): array;
+
+    /**
+     * @param array<string, mixed> $resolvedValues
+     *
+     * @return void
+     */
+    public function setResolvedValues(array $resolvedValues): void;
+
+    /**
+     * @param string $key
+     * @param mixed $value
+     *
+     * @return void
+     */
+    public function addResolvedValues(string $key, mixed $value);
+
+    /**
+     * @return array<\SprykerSdk\SdkContracts\Entity\MessageInterface>
+     */
+    public function getMessages(): array;
+
+    /**
+     * @param \SprykerSdk\SdkContracts\Entity\MessageInterface $message
+     *
+     * @return void
+     */
+    public function addMessage(MessageInterface $message);
```

Comment:
mb we can add return type here?

Reply:
Fixed

Reply:
update message structure array<string, MessageInterface> key will be id of command, also create errorCode properties as Messages. Errors should have array<string, int> structure

Reply:
update message structure array<string, MessageInterface> key will be id of command, also create errorCode properties as Messages. Errors should have array<string, int> structure